### PR TITLE
Mention tgswitch in recommended_tools.md

### PR DESCRIPTION
**Pull request by rhoboat**
- [x] Updated `tgenv` to `tgswitch` using the verbiage from https://blog.gruntwork.io/how-to-manage-multiple-versions-of-terragrunt-and-terraform-as-a-team-in-your-iac-project-da5b59209f2d. 
- [x] Updated description of `tfenv` based on this article, if that is okay. 
  - Taking a look at the sources for these tools, `pyenv` is partly written in `roff`, which is just for the manpages, I think. So maybe `tfenv` is actually inspired by both `rbenv` and `pyenv`, which are both mainly shell scripts.
- [x] Aimed to keep the em dashes.

### CODE REVIEW

File: _docs-sources/intro/environment-setup/recommended_tools.md [hunk 1]
```diff
@@ -9,8 +9,9 @@
 * [Terraform](https://www.terraform.io/downloads) – Terraform's declarative nature describes your infrastructure as code. If you're using Gruntwork's products, you're using Terraform.
 * [Terragrunt](https://terragrunt.gruntwork.io/docs/getting-started/install/) – Terragrunt is our layer on top of Terraform to enable a highly DRY code base.
 * [Terratest](https://github.com/gruntwork-io/terratest) – If you want to test your infrastructure, Terratest enables you to write those tests for manual testing or integration with a CI/CD pipeline.
-* [tfenv](https://github.com/tfutils/tfenv#installation) – `tfenv` is a version manager for Terraform, similar to `pyenv` for Python.
-* [tgenv](https://gitlab.com/claudiuskastner/tgenv) – `tgenv` is a version manager for Terragrunt
+* [tfenv](https://github.com/tfutils/tfenv#installation) – `tfenv` was inspired by a similar tool `rbenv` for Ruby versions. The tool is a set of bash scripts that provide a workflow for managing and using multiple versions of Terraform.
```
Comment (rhoboat) [inline _docs-sources/intro/environment-setup/recommended_tools.md:12]: Does this read better?

```suggestion
* [tfenv](https://github.com/tfutils/tfenv#installation) – `tfenv` is a set of bash scripts that provide a workflow for managing and using multiple versions of Terraform. It was inspired by similar tools `rbenv` for Ruby versions and `pyenv` for Python.
```

File: docs/intro/environment-setup/recommended_tools.md [hunk 2]
```diff
@@ -9,8 +9,9 @@
 * [Terraform](https://www.terraform.io/downloads) – Terraform's declarative nature describes your infrastructure as code. If you're using Gruntwork's products, you're using Terraform.
 * [Terragrunt](https://terragrunt.gruntwork.io/docs/getting-started/install/) – Terragrunt is our layer on top of Terraform to enable a highly DRY code base.
 * [Terratest](https://github.com/gruntwork-io/terratest) – If you want to test your infrastructure, Terratest enables you to write those tests for manual testing or integration with a CI/CD pipeline.
-* [tfenv](https://github.com/tfutils/tfenv#installation) – `tfenv` is a version manager for Terraform, similar to `pyenv` for Python.
-* [tgenv](https://gitlab.com/claudiuskastner/tgenv) – `tgenv` is a version manager for Terragrunt
+* [tfenv](https://github.com/tfutils/tfenv#installation) – `tfenv` was inspired by a similar tool `rbenv` for Ruby versions. The tool is a set of bash scripts that provide a workflow for managing and using multiple versions of Terraform.
```
Comment (rhoboat) [inline docs/intro/environment-setup/recommended_tools.md:12]: Does this read better?

```suggestion
* [tfenv](https://github.com/tfutils/tfenv#installation) – `tfenv` is a set of bash scripts that provide a workflow for managing and using multiple versions of Terraform. It was inspired by similar tools `rbenv` for Ruby versions and `pyenv` for Python.
```

File: _docs-sources/intro/environment-setup/recommended_tools.md [hunk 3]
```diff
@@ -9,8 +9,9 @@
...
+* [tgswitch](https://github.com/warrensbox/tgswitch) – `tgswitch` is a tool for versioning Terragrunt written in golang that offers similar features as `tfenv`, including managing the versions to use in a version file.
```
Comment (eak12913) [inline _docs-sources/intro/environment-setup/recommended_tools.md:13]:
```suggestion
* [tgswitch](https://github.com/warrensbox/tgswitch) – `tgswitch` is a tool for for managing and using multiple versions of Terragrunt. Written in golang, it offers similar features as `tfenv`, including managing the versions to use in a version file.
```

Comment (eak12913) [reply _docs-sources/intro/environment-setup/recommended_tools.md:13]: Not sure if that reads a little better?

**Review by eak12913 — COMMENTED** [review]
Thanks @rhoboat! Looks good overall. I had one small comment for your consideration. Just remember, if you merge the suggestions, you'll still want to pull locally and run `yarn regenerate:local` to get the output to regenerate into the `docs/` folder. Alternately you can just run `yarn start` to launch the site locally and that will also automatically regenerate.

File: _docs-sources/intro/environment-setup/recommended_tools.md (hunk 3 above)
Comment (rhoboat) [reply _docs-sources/intro/environment-setup/recommended_tools.md:13]: Thanks Eugene, I'll just make a slightly more improved suggestion

```suggestion
* [tgswitch](https://github.com/warrensbox/tgswitch) – `tgswitch` is a tool for managing and using multiple versions of Terragrunt. Written in golang, it offers similar features as `tfenv`, including managing the versions to use in a version file.
```

File: docs/intro/environment-setup/recommended_tools.md [hunk 4]
```diff
@@ -9,8 +9,9 @@
...
 * [Terragrunt](https://terragrunt.gruntwork.io/docs/getting-started/install/) – Terragrunt is our layer on top of Terraform to enable a highly DRY code base.
 * [Terratest](https://github.com/gruntwork-io/terratest) – If you want to test your infrastructure, Terratest enables you to write those tests for manual testing or integration with a CI/CD pipeline.
-* [tfenv](https://github.com/tfutils/tfenv#installation) – `tfenv` is a version manager for Terraform, similar to `pyenv` for Python.
-* [tgenv](https://gitlab.com/claudiuskastner/tgenv) – `tgenv` is a version manager for Terragrunt
+* [tfenv](https://github.com/tfutils/tfenv#installation) – `tfenv` is a set of bash scripts that provide a workflow for managing and using multiple versions of Terraform. It was inspired by similar tools `rbenv` for Ruby versions and `pyenv` for Python.
+* [tgswitch](https://github.com/warrensbox/tgswitch) – `tgswitch` is a tool for versioning Terragrunt written in golang that offers similar features as `tfenv`, including managing the versions to use in a version file.
```
Comment (rhoboat) [inline docs/intro/environment-setup/recommended_tools.md:13]:
```suggestion
* [tgswitch](https://github.com/warrensbox/tgswitch) – `tgswitch` is a tool for managing and using multiple versions of Terragrunt. Written in golang, it offers similar features as `tfenv`, including managing the versions to use in a version file.
```

File: _docs-sources/intro/environment-setup/recommended_tools.md [hunk 5]
```diff
@@ -9,8 +9,9 @@
...
 * [Terragrunt](https://terragrunt.gruntwork.io/docs/getting-started/install/) – Terragrunt is our layer on top of Terraform to enable a highly DRY code base.
 * [Terratest](https://github.com/gruntwork-io/terratest) – If you want to test your infrastructure, Terratest enables you to write those tests for manual testing or integration with a CI/CD pipeline.
-* [tfenv](https://github.com/tfutils/tfenv#installation) – `tfenv` is a version manager for Terraform, similar to `pyenv` for Python.
-* [tgenv](https://gitlab.com/claudiuskastner/tgenv) – `tgenv` is a version manager for Terragrunt
+* [tfenv](https://github.com/tfutils/tfenv#installation) – `tfenv` is a set of bash scripts that provide a workflow for managing and using multiple versions of Terraform. It was inspired by similar tools `rbenv` for Ruby versions and `pyenv` for Python.
+* [tgswitch](https://github.com/warrensbox/tgswitch) – `tgswitch` is a tool for managing and using multiple versions of Terragrunt. Written in golang, it offers similar features as `tfenv`, including managing the versions to use in a version file.
```
Comment (rhoboat) [inline _docs-sources/intro/environment-setup/recommended_tools.md:13]:
```suggestion
* [tgswitch](https://github.com/warrensbox/tgswitch#installation) – `tgswitch` is a tool for managing and using multiple versions of Terragrunt. Written in golang, it offers similar features as `tfenv`, including managing the versions to use in a version file.
```

File: docs/intro/environment-setup/recommended_tools.md [hunk 6]
```diff
@@ -9,8 +9,9 @@
...
 * [Terragrunt](https://terragrunt.gruntwork.io/docs/getting-started/install/) – Terragrunt is our layer on top of Terraform to enable a highly DRY code base.
 * [Terratest](https://github.com/gruntwork-io/terratest) – If you want to test your infrastructure, Terratest enables you to write those tests for manual testing or integration with a CI/CD pipeline.
-* [tfenv](https://github.com/tfutils/tfenv#installation) – `tfenv` is a version manager for Terraform, similar to `pyenv` for Python.
-* [tgenv](https://gitlab.com/claudiuskastner/tgenv) – `tgenv` is a version manager for Terragrunt
+* [tfenv](https://github.com/tfutils/tfenv#installation) – `tfenv` is a set of bash scripts that provide a workflow for managing and using multiple versions of Terraform. It was inspired by similar tools `rbenv` for Ruby versions and `pyenv` for Python.
+* [tgswitch](https://github.com/warrensbox/tgswitch) – `tgswitch` is a tool for managing and using multiple versions of Terragrunt. Written in golang, it offers similar features as `tfenv`, including managing the versions to use in a version file.
```
Comment (rhoboat) [inline docs/intro/environment-setup/recommended_tools.md:13]:
```suggestion
* [tgswitch](https://github.com/warrensbox/tgswitch#installation) – `tgswitch` is a tool for managing and using multiple versions of Terragrunt. Written in golang, it offers similar features as `tfenv`, including managing the versions to use in a version file.
```

**Review by eak12913 — APPROVED** [review]
LGTM!

**Pull request merged**